src/textscreens.cpp

```diff
@@ -377,14 +378,30 @@ void TextWindow::ShowGroupInfo() {
         Printf(true, " %Ft%s%E", s);
 
         bool one = (g->subtype == Group::Subtype::ONE_SIDED);
+        bool two = (g->subtype == Group::Subtype::TWO_SIDED);
+        bool skew = (g->subtype == Group::Subtype::ONE_SKEWED);
+        
+        if (g->type == Group::Type::EXTRUDE) {
+        Printf(false,
+            "%Ba   %f%Ls%Fd%s one-sided%E  "
+                  "%f%LS%Fd%s two-sided%E  "
+                  "%f%Lw%Fd%s skewed%E",
```

Comment:
I think it is better if this is called "perpendicular" and checked by default.

Reply:
> I think it is better if this is called "perpendicular" and checked by default.

Disagree. I think the checkbox should be "skewed" and unchecked by default. The normal/expected/natural behavior should not need to be enabled (even by default) and I kind of dislike the word "prependicular" vs "skewed". Maybe it's too many syllables...  ;-)